src/components/ui/app-footer.tsx

```diff
@@ -0,0 +1,26 @@
+"use client";
+
+import { useEffect, useState } from "react";
+
+export const AppFooter = () => {
+  const [version, setVersion] = useState("loading...");
+
+  useEffect(() => {
+    const fetchVersion = async () => {
+      try {
+        const res = await fetch("/api/v1/about");
+        const data = await res.json();
+        setVersion(data.data.version);
+      } catch (error) {
+        console.error("Error fetching version:", error);
+        setVersion("unknown");
+      }
+    };
+
+    fetchVersion();
+  }, []);
+
+  return (
+    <footer className="mt-8 border-t border-gray-300 pt-4 text-center text-sm text-gray-600">Version: {version}</footer>
```

Comment:
```suggestion
    <footer className="my-8 border-t border-gray-300 pt-4 text-center text-sm text-gray-600">Version: {version}</footer>
```
To have some place also on the bottom of element.

Reply:
done :)

Reply:
updated the API docs, pls let me know if it is ok like this